src/js/tooltip.js

```diff
@@ -305,13 +325,13 @@ class Tooltip {
 
 		switch (this.tooltipPosition) {
 			case 'above':
-				rect.top = this.target.top - (height + Tooltip.arrowDepth);
-				rect.left = this._getLeftFor('middle');
+				rect.top = this.target.top - height - Tooltip.arrowDepth;
+				rect.left = this._getLeftFor("middle");
```

Comment:
Double quote change is probably because I removed this line and then re-added it at a later date without thinking about quotes! I use them interchangeably - I know _I know_. I'll update them here so it's consistent with the rest of the file.